pkg/sensors/collection.go

```diff
@@ -36,6 +36,12 @@ func (s TracingPolicyState) ToTetragonState() tetragon.TracingPolicyState {
 	}
 }
 
+// collection_key is the unique key for sensors
+// this enables policies with the same name for different namespaces
+type collection_key struct {
```

Comment:
```suggestion
type collectionKey struct {
```
Go convention is to use camelCase for all symbols. Following conventions consistently makes the code easier to read, so let's follow it here.

Reply:
Updated. Thanks!